doc/sample_cookieblocklist.txt

```diff
@@ -554,6 +554,7 @@ where.com
 widgetserver.com
 wikidata.org
 wikimedia.org
+wikipedia.org
```

Comment:
Could we break this change out into its own pull request please? Here are some test sites:
- https://www.wikiwand.com/en/Enzo_Ferrari_(automobile) (from #1178)
- http://kevan.org/wikitext/
- https://www.wikiwand.com/en/Comparison_between_U.S._states_and_countries_by_GDP_(nominal)
- https://xefer.com/wikipedia